src/data/pi.lean

```diff
@@ -5,6 +5,7 @@ Authors: Simon Hudon, Patrick Massot, Eric Wieser
 -/
 import tactic.split_ifs
 import tactic.simpa
+import algebra.group.to_additive
 /-!
 # Theorems on pi types
```

Comment:
```suggestion
# Instances and theorems on pi types
```
And maybe the module doc should be updated too? (At the very least `Types` should not be capitalized and the sentence should end in a period.)